templates/default/node-exporter-consul.hcl.erb

```diff
@@ -2,7 +2,10 @@ service {
     id = "node_exporter/monitoring"
     name = "node_exporter"
     tags = [
-        "monitoring"
+        "monitoring",
```

Comment:
does hcl tolerate commas at the end is no more values?

Reply:
Yes it does